fmriprep/interfaces/cifti.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Handling connectivity
+~~~~~~~~~~~~~~~~~~~~~
+Combines FreeSurfer surfaces with subcortical volumes
+
+"""
+import os
+from glob import glob
+
+import nibabel as nb
+from nibabel import cifti2 as ci
+import numpy as np
+
+from niworkflows.nipype.interfaces.base import (
+    BaseInterfaceInputSpec, TraitedSpec, File, traits, isdefined,
+    SimpleInterface, Directory
+)
+
+from niworkflows.nipype.utils.filemanip import split_filename
+
+# CITFI structures with corresponding FS labels
+CIFTI_STRUCT_WITH_LABELS = {
+    # SURFACES
+    'CIFTI_STRUCTURE_CORTEX_LEFT': [],
+    'CIFTI_STRUCTURE_CORTEX_RIGHT': [],
+    # 'CIFTI_STRUCTURE_CORTEX': [],
+    # 'CIFTI_STRUCTURE_CEREBRAL_WHITE_MATTER_LEFT': [],
+    # 'CIFTI_STRUCTURE_CEREBRAL_WHITE_MATTER_RIGHT': [],
+
+    # SUBCORTICAL
+    'CIFTI_STRUCTURE_ACCUMBENS_LEFT': [26],
+    'CIFTI_STRUCTURE_ACCUMBENS_RIGHT': [58],
+    # 'CIFTI_STRUCTURE_ALL_WHITE_MATTER': [],
+    # 'CIFTI_STRUCTURE_ALL_GREY_MATTER': [],
+    'CIFTI_STRUCTURE_AMYGDALA_LEFT': [18],
+    'CIFTI_STRUCTURE_AMYGDALA_RIGHT': [54],
+    'CIFTI_STRUCTURE_BRAIN_STEM': [16],
+    'CIFTI_STRUCTURE_CAUDATE_LEFT': [11],
+    'CIFTI_STRUCTURE_CAUDATE_RIGHT': [50],
+    'CIFTI_STRUCTURE_CEREBELLAR_WHITE_MATTER_LEFT': [7],
+    'CIFTI_STRUCTURE_CEREBELLAR_WHITE_MATTER_RIGHT': [46],
+    # 'CIFTI_STRUCTURE_CEREBELLUM': [],
+    'CIFTI_STRUCTURE_CEREBELLUM_LEFT': [6],
+    'CIFTI_STRUCTURE_CEREBELLUM_RIGHT': [45],
+    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_LEFT': [28],
+    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_RIGHT': [60],
```

Comment:
or the diencephalon at present. i'm not sure these are labeled by the atlas.

Reply:
I used the labels provided on [mindboggle's page](http://mindboggle.readthedocs.io/en/latest/labels.html) - I'll remove these though

Reply:
ok you can leave these in place. do you have an example file on openmind that i can take a look at?